packages/chord-mark-themes/scss/themes/boxes.scss

```diff
@@ -0,0 +1,61 @@
+@import '../abstract';
+@import './common';
+
+.cmTheme-boxes {
+	@page {
```

Comment:
I'm not sure about this. With the other print theme, margins are managed directly by Chord Charts Studio and not the theme itself.

Reply:
I think handling it in Studio probably makes more sense. If we need a margin slider there, it could be added.